README.md

```diff
@@ -12,7 +12,32 @@ Once the internal systems are fully redeveloped this service will be either reti
 
 ## Getting started
 
-_TBD_
+To install, use:
+```
+make build
+```
+To run the tests, use:
+```
+make test
+```
+## Usage
+### Using Docker compose
+
+You can run this service using the docker compose file in [sdc-ci-upload-compose](https://github.com/ONSdigital/sdc-ci-upload-compose) and running `docker compose up -d`. This will run all the necessary services that
+it needs to communicate with. All you will need to do then is put a xlxs file inside the `Documents/ftp` directory and file should be put on the FTP.
+
+### Running Standalone
+
+You can run this service on its own but to do so, some changes will have to be made to environment variables such as `SEFT_FTP_PORT` and `SEFT_PUBLISHER_FTP_FOLDER`. This service also needs
+to communicate with an FTP server and a Rabbit queue, so these services will need to be created in order to upload files correctly. If you want to do so, your best option is to go to
+[sdc-ci-upload-compose](https://github.com/ONSdigital/sdc-ci-upload-compose) and run `docker-compose up -d sdx-seft-publisher-service`. This will build all the services that sdx-seft-publisher
+needs to run and pull a file from an FTP and put it on a Rabbit queue.
+
+To start the service, use the command:
+```
+python main.py
```

Comment:
Super minor sorry, but could you add a `make start` to the makefile of this repo with that command in it (and update the readme to reflect that).